apps/client/src/components/compounds/SetTarget/setTarget.module.scss

```diff
@@ -0,0 +1,68 @@
+.setTargetContainer {
+  background-color: var(--gray-50);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 43.437rem;
+  width: 28.375rem;
+  border-radius: 20px;
+  position: relative;
+  &CloseIcon {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+  }
+  &Text {
+    text-align: center;
+    padding-top: 0.2rem;
+    &Create {
+      margin-top: 0.5rem;
+      width: 20rem;
+    }
+  }
+  &Input {
+    width: 20rem;
+    margin-top: 1rem;
+    label {
+      margin-top: 1.5rem;
+    }
+    Button {
+      margin-top: 1.5rem;
+      width: 100%;
+    }
+    &Frequency {
+      position: relative;
+    }
+    &Arrow {
+      position: absolute;
+      bottom: 10%;
+      right: 5%;
+      z-index: 20;
+    }
+    &Hidden {
+      background: #fff;
+      text-align: start;
+      position: absolute;
+      right: -2%;
+      bottom: -108%;
+      z-index: 40;
+      line-height: 32.4px;
+      padding: 10px 10px;
+      border-radius: 10px;
+    }
+  }
+  @media (max-width: 425px) {
+    & {
```

Comment:
??

Reply:
i was adding responsiveness to it

& {: The "&" symbol refers to the parent selector. In this case, it represents the main container or element when the screen width matches the media query condition.

Reply:
you could do this instead:

```
@media (max-width: 425px){
   width: 100%;
      &Input {
        width: 100%;
      }
      &Text {
        &Create {
          width: 100%;
        }
      }
    }
    ```